public/components/GeoPlot.vue

```diff
@@ -284,6 +294,7 @@ export default Vue.extend({
       isClustering: false,
       isColoringByConfidence: false,
       confidenceIconClass: "confidence-icon",
+      isSatelliteView: false,
```

Comment:
to make it easier to read it would be nice to keep that list in an alphabetical order.

Reply:
Cleanup for this component is coming down the pipeline. This will get refactored at some point. The idea is to move all of the util out and move this list of variables into their own objects to make it much cleaner.